client/components/main/layouts.jade

```diff
@@ -13,6 +13,13 @@ template(name="userFormsLayout")
     h1.at-form-landing-logo
       img(src="{{pathFor '/wekan-logo.png'}}" alt="Wekan")
     +Template.dynamic(template=content)
+    div.at-form-lang
+      select.select-lang.js-userform-set-language
```

Comment:
BTW, with your proposal, will this section appear when user form in any state (sign-in, registration, pwd-forgot...)?